onnx/test/reference_evaluator_test.py

```diff
@@ -112,6 +120,18 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def skip_if_old_numpy_ver(fn):
+    @wraps(fn)
+    def wrapper(*args, **kwargs):
+        if version(npver) < version("1.21.5"):
+            raise unittest.SkipTest(
+                "op_dft and op_stft requires numpy >= 1.21.5"
+            ) from None
+        fn(*args, **kwargs)
+
+    return wrapper
```

Comment:
Would be nice to be able to specify the numpy version: I would use 

https://github.com/microsoft/onnxscript/blob/b6c5e3bb9729f939e979a41d8d724009df9f3b58/onnxscript/_internal/version_utils.py#L6-L13

and usage

https://github.com/microsoft/onnxscript/blob/b6c5e3bb9729f939e979a41d8d724009df9f3b58/onnxscript/function_libs/torch_lib/graph_building_test.py#L18

as reference

Reply:
restructured the code as suggested